source/localizable/tutorial/routes-and-templates.md

```diff
@@ -141,7 +141,7 @@ Here we will use the `{{link-to}}` helper in our code to link between routes:
     By building a property rental site, we can simultaneously imagine traveling
     AND building Ember applications.
   </p>
-  {{#link-to 'index' class="button"}}
+  {{#link-to 'contact' class="button"}}
```

Comment:
I think we do in fact mean to link to index here.  We want the user to "Get started" picking out a rental.